libs/coin-modules/coin-tezos/src/api/index.ts

```diff
@@ -109,10 +108,12 @@ async function estimate(transactionIntent: TransactionIntent): Promise<TezosFeeE
   } = await estimateFees({
     account: {
       address: transactionIntent.sender,
-      revealed: senderAccountInfo.revealed,
-      balance: BigInt(senderAccountInfo.balance),
+      revealed: senderAccountInfo.type === "user" ? senderAccountInfo.revealed : false,
```

Comment:
Boolean literals are quite unexpected
```suggestion
      revealed: senderAccountInfo.type === "user" && senderAccountInfo.revealed,
```